python/lsst/ts/xml/tables/m1m3/__init__.py

```diff
@@ -0,0 +1,35 @@
+# This file is part of ts_xml
+#
+# Developed for the LSST Data Management System.
```

Comment:
We have been using the Vera C. Rubin Observatory version for over two years now. Please update this line.